api-module-library/sharepoint/config/default.js

```diff
@@ -0,0 +1,30 @@
+
+if (process.env.NODE_ENV !== 'production') {
+    // Load values from .env
+    require('dotenv').config();
+}
+
+const apiModule = {
+    name: 'microsoft-sharepoint',
+    label: 'Microsoft SharePoint',
+    productUrl: 'https://microsoft.com/sharepoint',
+    apiDocs: 'https://developer.microsoft.com/en-us/graph/graph-explorer',
+    logoUrl: 'https://friggframework.org/assets/img/microsoft-sharepoint.jpeg',
+    categories: ['Sharing'],
+    description: 'SharePoint is a web-based collaborative platform that integrates natively with Microsoft 365 (previously, Microsoft Office)'
+};
```

Comment:
@seanspeaks  Not sure if that's best name for that variable, I guess values in here refer to the module itself?